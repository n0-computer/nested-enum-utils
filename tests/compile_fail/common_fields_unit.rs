use nested_enum_utils::common_fields;

#[common_fields({ x: u64 })]
enum Enum {
    A,
    B {},
}

fn main() {}
