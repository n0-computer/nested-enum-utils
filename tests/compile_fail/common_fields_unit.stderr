error: Expected named variants in enum
 --> tests/compile_fail/common_fields_unit.rs:3:1
  |
3 | #[common_fields({ x: u64 })]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `common_fields` (in Nightly builds, run with -Z macro-backtrace for more info)
